packages/worker/src/isolate-builder.ts

```diff
@@ -66,15 +67,23 @@ export class WorkflowIsolateBuilder {
     let code = dedent`
       const init = require('@temporalio/workflow/lib/init');
       const internals = require('@temporalio/workflow/lib/internals');
-      init.overrideGlobals();
       internals.state.activityDefaults = ${JSON.stringify(this.activityDefaults)};
-
+      init.overrideGlobals();
```

Comment:
Still not loving that this code is just a string. Seems like other stuff might keep getting added here and it may be worth finding a different way.